README.md

```diff
@@ -61,6 +61,51 @@ You can also always find the full command-line options that are available with
 $ cargo fuzz --help
 ```
 
+## Generating code coverage information
+
+Use the `--coverage` option to generate precise
+[source-based code coverage](https://blog.rust-lang.org/inside-rust/2020/11/12/source-based-code-coverage.html)
+information:
+```
+$ cargo fuzz run --coverage <coverage data output file> <target>
+```
+This compiles your project using the `-Zinstrument-coverage` Rust compiler flag and generates coverage data in the
+specified file. This file can be used to generate coverage reports and visualize code-coverage information.
```

Comment:
This appears to be somewhat inaccurate. At least as per the test added below the coverage is saved to `coverage_output.prof` despite `--coverage coverage_output` being specified as the argument.

Seems to me that this detail is important enough that its worth finding a way to describe this behaviour precisely.